uk.co.compucorp.civicrm.tasksassignments/js/dist/tasks-assignments.min.js

```diff
@@ -8408,6 +8409,34 @@ define('tasks-assignments/controllers/modal/modal-document',[
         AssignmentService.updateCache(obj);
       };
 
+      /**
+       * Searches the list of assignments for the given target contact
+       *
+       * @param  {string | integer} targetContactId
+       */
+      $scope.cacheTargetContactAssignments = function (targetContactId) {
```

Comment:
I have used the `cache` prefix as the resulted assignments will be  assigned to `$scope.assignments`. But as you pointed `$scope.searchContactAssignments ` sounds better. What do you suggest?